core-bundle/src/Resources/views/Backend/be_undo_label.html.twig

```diff
@@ -0,0 +1,35 @@
+{% trans_default_domain 'contao_default' %}
```

Comment:
Would't it make more sense to use `contao_tl_undo` as the default domain? :upside_down_face:

Reply:
See 9f4534f4e182fa59f6c16172dd8bf38b1273e90f